ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -3116,7 +3132,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1061</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.9</ns1:Doc_Sect>

```

Comment:
Incorrect section number. It should be 2.2.4.26

Reply:
Incorrect section number issue with below RS:
MS-OXWSCORE_R1359
MS-OXWSCORE_R1360
MS-OXWSCORE_R1361
MS-OXWSCORE_R1364